gamepad/gamepad/publisher.py

```diff
@@ -0,0 +1,206 @@
+import rclpy, os
+from rclpy.node import Node
+from std_msgs.msg import String
+from inputs import devices
+from inputs import get_gamepad
+infile_path = "/dev/input/js0"
+
+
+class MinimalPublisher(Node):
+
+    def __init__(self):
+        super().__init__('minimal_publisher')
+        self.colorbuttons = self.create_publisher(String, 'abxy', 10) 
```

Comment:
I think it would be better to have one publisher that includes all of this information, rather than publishing to multiple separate topics